depends/packages/boost.mk

```diff
@@ -1,7 +1,7 @@
 package=boost
 $(package)_version=1_70_0
 $(package)_download_path=https://dl.bintray.com/boostorg/release/1.70.0/source/
-$(package)_file_name=$(package)_$($(package)_version).tar.bz2
+$(package)_file_name=boost_$($(package)_version).tar.bz2
```

Comment:
Perhaps it's cleaner to inherit from the `native` package like you did for `capnp` and `libmultiprocess`? Maybe I'm missing something here though.

Reply:
> Perhaps it's cleaner to inherit from the `native` package like you did for `capnp` and `libmultiprocess`? Maybe I'm missing something here though.

That'd be another approach. I wouldn't mind adding that, but it would make the change to the boost package here a little bigger and less minimal than actually needed for the PR.

Also, I do think the change here is an improvement by itself. It seems janky to me tie our local package name to the convention the upstream project is using for download urls.

Reply:
+1. It's fine when it's convenient shorthand or more clear, but no need to keep them tied if it makes anything harder.